### ISSUE TITLE
Current repository state

### ISSUE THREAD

**Issue by crayonwow** [OvyFlash/telegram-bot-api#34]
Hello
I've been looking around and found this fork. At this moment this fork have few problems that obviously been inherited from parent repo. In details:

1. Outdated README.md
2. Outdated go.mod (go version)
3. Outdated GitHub workflows
4. Lack of proper versioning
5. Does not supports `go get`
6. Outdated docs
7. Outdated practices (for example logginig)

Is there any desire to improve the repo in this directions? I case it does i can offer my help

**Comment by iamwavecut** [comment]
@crayonwow Well it's official

**Comment by kuchaguangjie** [comment]
There is no tag or release, I saw branches like `7.10`, so that's the suggested way to use this repo while keep track of the version?

**Comment by iamwavecut** [comment]
@kuchaguangjie for now, just use /v6 package name, but it may be a subject to change in the future
```shell
go get -u github.com/OvyFlash/telegram-bot-api/v6
```

**Comment by iamwavecut** [comment]
@OvyFlash consider adding some code owners from community, to be able to merge PR's and stuff.

**Comment by kuchaguangjie** [comment]
> @OvyFlash consider adding some code owners from community, to be able to merge PR's and stuff.

Yeah, tag is needed, @OvyFlash if you don't have time, please add people like @iamwavecut as code owner.

**Comment by kuchaguangjie** [comment]
When I try to use branch `7.10` via:
> go get github.com/OvyFlash/telegram-bot-api@bot-api-7.10

get error:
```
go: github.com/OvyFlash/telegram-bot-api@bot-api-7.10: github.com/OvyFlash/telegram-bot-api@v0.0.0-20241013102559-5fd45917caa1: invalid version: go.mod has post-v0 module path "github.com/go-telegram-bot-api/telegram-bot-api/v5" at revision 5fd45917caa1
```
So, is the 7.10 version ready? If so how to use it, since there is no tag or release at all ?

**Comment by iamwavecut** [comment]
@kuchaguangjie 
for now just use /v6 package module, it's the current working branch, so should be ok

**Comment by kuchaguangjie** [comment]
For now, I will use this one: https://github.com/mymmrac/telego
It's really easy to use, and its api is auto generated from telegram bot api doc (https://telego.pixelbox.dev/docs/introduction/overview/#how-one-to-one-implementation-achieved).

**Comment by umputun** [comment]
> When I try to use branch `7.10` via:
> 
> > go get github.com/OvyFlash/telegram-bot-api@bot-api-7.10
> 
> get error:
> 
> ```
> go: github.com/OvyFlash/telegram-bot-api@bot-api-7.10: github.com/OvyFlash/telegram-bot-api@v0.0.0-20241013102559-5fd45917caa1: invalid version: go.mod has post-v0 module path "github.com/go-telegram-bot-api/telegram-bot-api/v5" at revision 5fd45917caa1
> ```
> 
> So, is the 7.10 version ready? If so how to use it, since there is no tag or release at all ?

`go get  github.com/OvyFlash/telegram-bot-api/v6` will do until @OvyFlash considers this version stable and add `v6.x.y` tag

**Comment by iamwavecut** [comment]
Please check the updated README.md for details, but be noted, that versioned tags are highly discouraged due to the simple fact, that Telegram Bot API does not support versioning itself. I will maintain tagging to follow new API updatesm but it would be wise of you to not use pinned version in your upstream projects!

Replace 
```
github.com/OvyFlash/telegram-bot-api/v6
```
with
```
github.com/OvyFlash/telegram-bot-api
```
in `*.go` files. 

Replace 
```
github.com/OvyFlash/telegram-bot-api/v6 v6.0.0-*
```
with
```
github.com/OvyFlash/telegram-bot-api v0.0.0-20241107134340-8afd2aca51ec
```
in your `go.mod`, and run `go mod tidy`

**Comment by crayonwow** [comment]
damn, i forgot about this
i'll just push what i already have
it's not done yet and probably outdated

**Comment by monoflash** [comment]
If everyone who writes here knows Russian, why communicate in English?
At least take an example from the Chinese, they are not shy about the fact that they are Chinese and boldly communicate in a language that is convenient to them.

Regarding the repository. The original repository from which the first fork was made has been dead for several years!
I somehow spent time and collected the most useful of all the forks and PR, but it's difficult to maintain all this alone. Therefore, it may be worth untying this fork from the original fork and changing everything here, since this fork is alive.

**I'm all for @crayonwow's suggestion!**

At least the original repository can't be resurrected for sure, everything is outdated there and RP has not been accepted for 3+ years at all, my PR was hanging there for a year, then I deleted it.
